readme.md

```diff
@@ -111,7 +111,7 @@ In order to run CANU, both python3 and pip3 need to be installed.
   ```bash
     python3 -m venv .venv
     source ./.venv/bin/activate
-    pip3 install ./canu
+    pip3 install .
```

Comment:
I like the thought behind this.. 

The original way assumed you `git clone` canu which will create a `canu` directory where you run the command.  The venv is then created parallel to that:  so `canu/` and `.venv/` are at the same directory level.  Your proposal is that the `.venv/` directory is inside the `canu/` directory.  

There's nothing wrong with either way.  The real problem here, as I see it, is that we allow the ambiguity in the first place.  Perhaps we should have a new `git clone` section that then leaves you in the appropriate place? Personally I prefer having the directories parallel as they currently exist in the docs, but ymmv.  The fact you couldn't direct follow the installation directions is indeed a problem.

Reply:
I appreciate what you said about git clone but ./.venv line above implies you're in the repo directory already.

Reply:
FYI my actual needed commands to get canu to run properly at this time is this:
```
python3 -m venv .venv
source ./.venv/bin/activate
.venv/bin/python3.9 -m pip install --upgrade pip
pip3 install wheel
pip3 install .
```